src/boot/efi/util.h

```diff
@@ -32,11 +32,12 @@ CHAR16 *stra_to_path(CHAR8 *stra);
 CHAR16 *stra_to_str(CHAR8 *stra);
 
 EFI_STATUS file_read(EFI_FILE_HANDLE dir, CHAR16 *name, UINTN off, UINTN size, CHAR8 **content, UINTN *content_size);
-#endif
 
 static inline void FreePoolp(void *p) {
         FreePool(*(void**) p);
 }
 
 #define _cleanup_(x) __attribute__((cleanup(x)))
 #define _cleanup_freepool_ _cleanup_(FreePoolp)
+
+#endif
```

Comment:
It's consistent with the rest of the headers in `src/boot/efi` and works totally fine, but it'd probably make sense to start using `#pragma once` sometime in the future.

Reply:
Yep, we should.